enginetest/queries.go

```diff
@@ -1639,23 +1639,98 @@ var QueryTests = []QueryTest{
 	},
 	{
 		`SHOW COLLATION`,
-		[]sql.Row{{"utf8mb4_0900_ai_ci", "utf8mb4", int64(1), "Yes", "Yes", int64(1), "PAD SPACE"}},
+		[]sql.Row{
+			{
+				sql.Collation_binary.String(),
+				"binary",
+				sql.CollationToMySQLVals[sql.Collation_binary].ID,
+				sql.CollationToMySQLVals[sql.Collation_binary].IsDefault,
+				sql.CollationToMySQLVals[sql.Collation_binary].IsCompiled,
+				sql.CollationToMySQLVals[sql.Collation_binary].SortLen,
+				sql.CollationToMySQLVals[sql.Collation_binary].PadSpace,
+			},
+			{
+				sql.Collation_utf8_general_ci.String(),
+				"utf8mb3",
```

Comment:
mysql reports this as "utf8"

Reply:
Darn it Zach, `utf8mb3` and `utf8` are aliases. `utf8` is deprecated so we're explicit in using `utf8mb3` if you pass in `utf8`.